Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+@Library('pipeline-library')
+import com.genesys.jenkins.Service
+
+def notifications = null
+String[] mailingList = [
+  "Brian.Dupuis@genesys.com"
```

Comment:
can I add myself to the mailing list?

Reply:
I'm actually going to test this script first, get it working, _then_ add others into the mailing list. I just don't want everyone to be getting spammed with failures until I get it working.

Reply:
I wonder if there's a way to get a mailing list injected by the jenkins pipeline instead of hard-coding a list of emails in the Jenkinsfile?

Reply:
That's the thing with Jenkinsfiles, they pretty much do all their own stuff rather than having much configuration.

Reply:
@crumpf thought on it more and found parameters... let's give that a shot. I've adjusted to attempt using a parameter for it. Good idea!